### PR TITLE
Added stray-marker tracking functionality.

### PR DESCRIPTION
**Pull request by TomShkurti**
Master branch of our fork includes documentation and additional .rom files in compliance with in-house standards. Lightweight branch is just stray marker tracking.

### CODE REVIEW

File: dependencies.rosinstall [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# NOTE: This is an example source dependency, remove it before adding the
+# dependencies.rosinstall file to your repository.
+#
+# The 'Serial Communication Library' utility is a cross-platform library for interfacing with rs-232 serial like ports written in C++. It provides a modern C++ interface with a workflow designed to look and feel like PySerial, but with the speed and control provided by C++.
+
+- git:
+    local-name: wjwwood/serial
+    uri: https://github.com/wjwwood/serial
```
Comment (ahoarau) [inline dependencies.rosinstall]: is this file necessary ? dependencies can be retrieved with rosdep

Comment (TomShkurti) [reply dependencies.rosinstall]: One of our internal files (for Travis) that didn't get taken out of Lightweight.

File: CMakeLists.txt [hunk 2]
```diff
@@ -29,23 +37,22 @@ include_directories(
...
   ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
   LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
   RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
 )
-
-install(DIRECTORY include/${PROJECT_NAME}/
- DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
- FILES_MATCHING PATTERN "*.h"
- PATTERN ".svn" EXCLUDE
-)
```
Comment (ahoarau) [inline CMakeLists.txt:51]: Why did you remove this ?

**Comment by ahoarau** [comment]
Thanks for the pr, but I need this original ROM files to be there (there are deleted in this pr)

**Comment by ahoarau** [comment]
Also, why didn't you use a geometry_msg/PointStamped ?
